axol/axol.py

```diff
@@ -714,46 +727,112 @@ def femka(message):
+                                           "\nНАПИШИ ТЕКСТ ИЛИ ОТПРАВЬ <s>NUDES</s> ФОТОЧКУ, ГОЛОС, КРУГЛОВИДЕО: "
+                                           "Я ЗАБОТЛИВО УПАКУЮ И ДОСТАВЛЮ ЧТО УГОДНО. В ЧАТИК И АНОНИМНО ;) "
+                                           "\n\n ЕСЛИ НЕ ХОЧЕШЬ НИЧЕГО ОТПРАВЛЯТЬ, ПРОСТО НЕ ОТВЕЧАЙ НА ЭТО СООБЩЕНИЕ",
+                     reply_markup=force_send_card, parse_mode="HTML")
 
 
+@bot.message_handler(
+    content_types=['text', 'sticker', 'photo', 'video', 'video_note', 'voice', 'audio', 'document', 'animation'],
+    func=lambda message: check_valentine(message))
+def love_send(message):
+    hashtag = "#валентинка"
+    for chat in love_chats:
+        if message.content_type == 'text':
+            bot.send_message(chat, hashtag + "\n" + message.text)
+        else:
+            bot.send_message(chat, hashtag + "\n")
+            if message.content_type == 'sticker':
+                bot.send_sticker(chat, message.sticker.file_id)
+            if message.content_type == 'photo':
+                bot.send_photo(chat, message.photo[0].file_id, caption=message.caption)
+            if message.content_type == 'video':
+                bot.send_video(chat, message.video, caption=message.caption)
+            if message.content_type == 'video_note':
+                bot.send_video_note(chat, message.video_note.file_id)
+            if message.content_type == 'voice':
+                bot.send_voice(chat, message.voice.file_id)
+            if message.content_type == 'audio':
+                bot.send_audio(chat, message.audio.file_id, caption=message.caption)
+            if message.content_type == 'document':
+                bot.send_document(chat, message.document.file_id, caption=message.caption)
+            if message.content_type == 'animation':
+                bot.send_animation(chat, message.animation.file_id, caption=message.caption)
+
+
+def check_valentine(message):
+    return message.reply_to_message \
+           and message.reply_to_message.from_user.id == bot.get_me().id \
+           and message.from_user.id == message.chat.id
+
+
+@bot.callback_query_handler(func=lambda call: call.data == "task_data")
+def task_callback(call):
+    player = findplayer(call.from_user)
+    answer = love_task_info(player)
+    if not answer:
+        answer = "ПОКА МНЕ НЕЧЕГО ТЕБЕ ПОКАЗАТЬ"
+    bot.send_message(call.message.chat.id, answer)
+
+
+# love commands for root
 @bot.message_handler(commands=["love_set"])
-def love_set(message):
+def love_set(self):
+    if check_love_tasks_exist():  # todo setting love tasks again requires explicitly clear already existed tasks
+        return
     players_in_love = []
     for player in active_players:
         if player.islove:
             try:
-                status = bot.get_chat_member(vip_chat_id, player.user.id)
+                status = bot.get_chat_member(love_chats, player.user.id)
```

Comment:
Is this function works properly with list with single chat instead of single id?
Would it function work if we add more chats into love_chats? If no, then love_chats should be renamed to love_chat

Reply:
Replaced list of chat with a single chat, because we don't support multi-chat mode now.